pkg/collector/corechecks/snmp/config.go

```diff
@@ -218,6 +227,19 @@ func buildConfig(rawInstance integration.Data, rawInitConfig integration.Data) (
 		c.timeout = int(instance.Timeout)
 	}
 
+	if instance.ExtraMinCollectionInterval != 0 {
+		c.minCollectionInterval = time.Duration(instance.ExtraMinCollectionInterval) * time.Second
+	} else if instance.MinCollectionInterval != 0 {
+		c.minCollectionInterval = time.Duration(instance.MinCollectionInterval) * time.Second
+	} else if initConfig.MinCollectionInterval != 0 {
+		c.minCollectionInterval = time.Duration(initConfig.MinCollectionInterval) * time.Second
+	} else {
+		c.minCollectionInterval = defaults.DefaultCheckInterval
+	}
+	if c.minCollectionInterval < 0 {
+		return snmpConfig{}, fmt.Errorf("min collection interval must be > 0, but got: %v", c.minCollectionInterval.Seconds())
```

Comment:
TODO: check backward compatible, check if agent min_collection_interval impl is returning error if value is 0